HtmlCssSample.css

```diff
@@ -0,0 +1,41 @@
+body
```

Comment:
Follow the below standards while writing the css
Use lowercase letters and dashes (-)
- Avoid using IDs (#) for styling
- Use meaningful and descriptive names
- Keep names short and concise
- Use namespaces (e.g. .header__logo) for nested elements
- Use prefixes (e.g. .btn-) for variations

Reply:
Resolved